/*_ <!base=example!> <!pagitterStoresCreate=example!> <!flavour=delicious!> example.js _*/

function(){
	return "<!flavour!>"
}

/*_ <!pagitterStoresEnd=example!>  example2.js _*/

function(){
	return "<!flavour!>"
}
function(){
	return "<!flavour!>"
}